api/services/cartService.js

```diff
@@ -0,0 +1,25 @@
+require("dotenv").config();
+const cartDao = require("../models/cartDao");
+
+
+const getCart = async (userId) => {
+  const cartList = await cartDao.getCart(userId);
+  return await cartList;
+};
+
+
+const addCart = async (itemId, optionId) => {
+  const added = await cartDao.addCart(itemId, optionId);
+  return await added;
+};
+
+const deleteCart = async (cartId) => {
+  return await cartDao.deleteCart(cartId);
+};
+
+
+module.exports = {
```

Comment:
-  공백을 주 때는 한 칸만 주세요.
```suggestion
require("dotenv").config();
const cartDao = require("../models/cartDao");

const getCart = async (userId) => {
  const cartList = await cartDao.getCart(userId);
  return await cartList;
};

const addCart = async (itemId, optionId) => {
  const added = await cartDao.addCart(itemId, optionId);
  return await added;
};

const deleteCart = async (cartId) => {
  return await cartDao.deleteCart(cartId);
};

module.exports = {
```